public/app/plugins/panel/alertlist/UnifiedAlertList.tsx

```diff
@@ -60,14 +76,86 @@ export function UnifiedAlertList(props: PanelProps<UnifiedAlertListOptions>) {
     dashboard = getDashboardSrv().getCurrent();
   });
 
+  const stateList = useMemo(() => getStateList(props.options.stateFilter), [props.options.stateFilter]);
+
+  function getMatcherListFromString(matcher: string) {
```

Comment:
Maybe we should extract this helper function to another file containing functions that can do both convertions from JSONstring and to JSONstring